tests/unit_tests/libs_tests/GameEntityManipulation_tests/CreateEntitiesFunctions_tests/CreateBackGround_tests.cpp

```diff
@@ -0,0 +1,21 @@
+/*
+** EPITECH PROJECT, 2022
+** R-Type
+** File description:
+** CreateBackGround_tests
+*/
+
+#include <criterion/criterion.h>
+#include "World/World.hpp"
+#include "GameEntityManipulation/CreateEntitiesFunctions/CreateBackground.hpp"
+
+using namespace ecs;
+
+Test(CreateBackGround_test, CreateBackGround)
+{
+    World world(1);
+
+    std::size_t id_new_entity = createNewBackGround(world, 10, 10, 1, 10, 10, 1, 1);
+
+    cr_assert_eq(1, id_new_entity);
```

Comment:
Why don't you check any components ?

Reply:
Because it's already test in world_tests with function add component